src/data/matrix/kronecker.lean

```diff
@@ -122,6 +122,30 @@ end
   kronecker_map f (1 : matrix m m α) (1 : matrix n n β) = 1 :=
 (kronecker_map_diagonal_diagonal _ hf₁ hf₂ _ _).trans $ by simp only [hf₃, diagonal_one]
 
+lemma kronecker_map_reindex_left (f : α → β → γ) (el : l ≃ l') (em : m ≃ m') (M : matrix l m α)
+  (N : matrix n n' β) : matrix.kronecker_map f (matrix.reindex el em M) N =
+  matrix.reindex (el.prod_congr (equiv.refl _)) (em.prod_congr (equiv.refl _))
+  (matrix.kronecker_map f M N) := by { ext ⟨i, i'⟩ ⟨j, j'⟩, refl }
+
+lemma kronecker_map_reindex_right (f : α → β → γ) (em : m ≃ m') (en : n ≃ n') (M : matrix l l' α)
+  (N : matrix m n β) : matrix.kronecker_map f M (matrix.reindex em en N) =
+  matrix.reindex ((equiv.refl _).prod_congr em) ((equiv.refl _).prod_congr en)
+    (matrix.kronecker_map f M N) := by { ext ⟨i, i'⟩ ⟨j, j'⟩, refl }
```

Comment:
You can remove all the `matrix.` prefixes here.

Can you add the `simp` version mentioned in the zulip chat too, which has no refls on the LHS?

Reply:
Sure, I forgot them when importing from LTE. Concerning the `simp` version, do you mean this
```
(M ⊗ₖ N).minor (e1.prod_congr e2) (e3.prod_congr e4) = (M.minor e1 e3) ⊗ₖ (N.minor e1 e3)
```
or something else?

Reply:
Yes, that's the one I mean, although lets write it with `reindex` since that's what you have here too.

Reply:
OK, I inserted the lemma. As you see, the two `left` and `right` variants have two proofs, one is the "old" one, and the other is using the new lemma. They are slightly more verbose but single-tactic. I don't know which one to prefer.